src/instructlab/sdg/generate_data.py

```diff
@@ -284,6 +289,7 @@ def generate_data(
     client: Optional[openai.OpenAI] = None,
     pipeline: Optional[str] = "simple",
     batch_size: Optional[int] = None,
+    checkpoint_dir: Optional[str] = None,
```

Comment:
Are we expecting this to be a CLI arg?

Reply:
I thought it would be, a user would need to be aware checkpoints are being used and control if they want it or not  (checkpoint_dir=None)
https://github.com/instructlab/instructlab/pull/1889